src/components/BookList.js

```diff
@@ -1,29 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const BookList = ({ title, author }) => (
-  <li>
-    <div>
-      <h1>{title}</h1>
-      <h2>{author}</h2>
-      <ul>
-        <li>
-          <button type="button">Comments</button>
-        </li>
-        <li>
-          <button type="button">Remove</button>
-        </li>
-        <li>
-          <button type="button">Edit</button>
-        </li>
-      </ul>
-    </div>
-  </li>
-);
+function BookList({
+  title, author, deletebook, id,
+}) {
+  return (
+    <li>
+      <div>
+        <h1>{title}</h1>
+        <h2>{author}</h2>
+        <ul>
+          <li>
+            <button type="button">Comments</button>
+          </li>
+          <li>
+            <button type="button" onClick={() => deletebook(id)}>Remove</button>
```

Comment:
- [ ] Kindly create a function in your component let's say `removeBook` that takes the id from book data and dispatches the `deletebook` action creator with the id as its argument, instead of calling the `deletebook` directly as you have done.